### ISSUE TITLE
MySQL 8 issue

### ISSUE THREAD

**Issue by sonicpunk** [Jako/CrossContextsSettings#1]
We are developing with MySQL 8 and seeing this error in the modx logs:

```
Array
(
    [0] => 42000
    [1] => 1064
    [2] => You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'asc ORDER BY `modContextSetting`.`key` ASC LIMIT 10' at line 1
)
```

**Comment by sonicpunk** [comment]
The CMP is not able to load context settings.

Tried the latest version 1.2.0 as well and found no differences.

**Comment by Jako** [comment]
This is fixed with 1.2.1
